cfgov/jinja2/v1/_layouts/base-common.html

```diff
@@ -192,6 +184,8 @@
       https://github.com/Modernizr/Modernizr/tree/master/feature-detects/es5
     #}
     <script>
+        {% include '/js/modernizr.min.js' %}
+
         var modernizr = window.Modernizr;
         if ( !( typeof modernizr !== 'undefined' &&
              modernizr.es5array &&
```

Comment:
Oops, looks like the IE9 script removal changes added in https://github.com/cfpb/cfgov-refresh/pull/3693/files were overridden with the old template when `base-common.html` was added. Do you want to get those changes here or is it easier if I open a PR?

Reply:
good catch, I'll add it back.

Reply:
I went ahead and opened a PR https://github.com/cfpb/cfgov-refresh/pull/3900 for it, since it's really not overlapping with your changes here.

Reply:
ok, I updated the PR to load Modernizr form an external script. Let's hash that out in your PR though.

Reply:
So based on https://github.com/Modernizr/Modernizr/issues/878#issuecomment-41448059 it should be inline, right?

Reply:
yep, fixed in https://github.com/cfpb/cfgov-refresh/pull/3893/commits/941b5ede22be576f62d7c8fb86e79b3a058c18e8